### PR TITLE
Modified RAV4 params

### PR DESCRIPTION
**Pull request by reyescuba**
Seems like every RAV4 it's running this parameters on there RAV4 , and also OP V5.10 improved when I made this changes smoothly, thanks

### CODE REVIEW

**Comment by arne182** [comment]
I can confirm this

**Comment by jsh348** [comment]
On Line 97:
``` python
ret.mass = 3650 * CV.LB_TO_KG + std_cargo           # original comma branch
ret.mass = 4100./2.205 + std_cargo                  # arne's (and many other RAV4 forks) branch
ret.mass = 4100./2.205 * CV.LB_TO_KG + std_cargo    # this pull request
ret.mass = 4100 * CV.LB_TO_KG + std_cargo           # I think this is the correct way to do it
```
[link to arne's file](https://github.com/arne182/openpilot/blob/3d1ace6d85924a25bedfe0c1e57dfd0ae762ed9d/selfdrive/car/toyota/interface.py#L126)

@arne182 correct me if I'm wrong, but I believe `./2.205` is equivalent to `* CV.LB_TO_KG `, and there is no need to do it twice.

@heatnation I should have been more clear on discord, sorry...

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -92,11 +92,11 @@ def get_params(candidate, fingerprint):
       stop_and_go = True if (candidate in CAR.RAV4H) else False
       ret.safetyParam = 73  # see conversion factor for STEER_TORQUE_EPS in dbc file
       ret.wheelbase = 2.65
-      ret.steerRatio = 16.30   # 14.5 is spec end-to-end
+      ret.steerRatio = 15.50   # 14.5 is spec end-to-end
       tire_stiffness_factor = 0.5533
-      ret.mass = 3650 * CV.LB_TO_KG + std_cargo  # mean between normal and hybrid
-      ret.steerKpV, ret.steerKiV = [[0.6], [0.05]]
-      ret.steerKf = 0.00006   # full torque for 10 deg at 80mph means 0.00007818594
+      ret.mass = 4100./2.205 * CV.LB_TO_KG + std_cargo  # mean between normal and hybrid
```
Comment (legonigel) [inline selfdrive/car/toyota/interface.py]: This seems wrong that you divide the weight in lbs by 2.205 then multiply by LB_TO_KG. Please fix the mass calculation.

Comment (heatnation) [reply selfdrive/car/toyota/interface.py]: Ok,just leave the Weight as default 3650 and change everything else I changed,thanks

**Pull request closed without merging**